data/mp/stats/research.json

```diff
@@ -7540,10 +7549,6 @@
 		"name": "Assault Gun",
 		"redComponents": [
 			"CyborgChaingun",
```

Comment:
You also redacted the CyborgChaingun in the Heavy Machinegun so decide which one to remove.

Reply:
Thanks.
fixed